elfi/env.py

```diff
@@ -1,12 +1,15 @@
 from collections import defaultdict
 import socket
+import logging
 
 from distributed import Client, LocalCluster
 from elfi.inference_task import InferenceTask
 
 _globals = defaultdict(lambda: None)
 _whitelist = ["client", "inference_task"]
 
+logging.getLogger('tornado').setLevel(logging.WARNING)
```

Comment:
Swapped order (though I don't see the point).

So now that the level is WARNING, we do get the warnings. If it would be DEBUG, then we would get those and a lot of useless spam.